app/controllers/plans_controller.rb

```diff
@@ -1,3 +1,5 @@
+require 'active_support/core_ext/object/try'
+
```

Comment:
Did you add this to deal with an error? I think this should be included already.